### PR TITLE
add volk/1.2.182

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **volk/1.2.182**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by Croydon — DISMISSED** [review]

File: recipes/volk/all/conandata.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.182":
+    url: "https://github.com/zeux/volk/archive/1.2.170.tar.gz"
+    sha256: "01f37fc0c904afd2eb8060cada73f903ab1cdae87c7c24e77585ca7fb9bafae4"
```
Comment (SpaceIm) [inline recipes/volk/all/conandata.yml:4]: it doesn't match, for a good reason: there is no volk 1.2.182.

Comment (AndreyMlashkin) [reply recipes/volk/all/conandata.yml]: Fixed :)

**Comment by AndreyMlashkin** [comment]
Is it a CI issue?
volk/1.2.170: Retrieving package 32708a3a980a05cdaba6f4900f1ce74b7b6aecae from remote 'c3i_PR-7181' 
ERROR: Binary package not found: 'volk/1.2.170:32708a3a980a05cdaba6f4900f1ce74b7b6aecae'

File: recipes/volk/all/conandata.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.2.182":
```
Comment (madebr) [inline recipes/volk/all/conandata.yml:2]: I don't know about CI, but there is 1.2.190 now.
https://github.com/zeux/volk/releases/tag/1.2.190

**Comment by madebr** [comment]
While you're working on volk, could you please look at the hooks warning(s)?
```
[HOOK - conan-center.py] pre_export(): WARN: [TOOLS CROSS BUILDING (KB-H062)] The 'tools.cross_building(self.settings)' syntax in /home/conan/w/cci_PR-7181/recipes/volk/all/test_package/conanfile.py may not work correctly in some scenarios. Consider using tools.cross_building(self).
```

**Comment by madebr** [comment]
> Is it a CI issue?
> volk/1.2.170: Retrieving package 32708a3a980a05cdaba6f4900f1ce74b7b6aecae from remote 'c3i_PR-7181'
> ERROR: Binary package not found: 'volk/1.2.170:32708a3a980a05cdaba6f4900f1ce74b7b6aecae'

@jgsogo 
This looks like a c3i issue.
The following error is emitted by the test package:
```
ERROR: Binary package not found: 'volk/1.2.170:32708a3a980a05cdaba6f4900f1ce74b7b6aecae'
```

See https://github.com/conan-io/conan-center-index/pull/7181#issuecomment-913825594

**Comment by AndreyMlashkin** [comment]
opened a newer one
https://github.com/conan-io/conan-center-index/pull/7192

**Pull request closed without merging**